docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -124,38 +149,31 @@ Now that we have a table with the desired three partitions, the final step is to
 
 ![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-2.png)
 
-
 First, we pin the data of the EU partition `transactions_eu` to live across three zones of the Europe (Frankfurt) region `eu-central-1` as shown below.
 
-
-```
+```sh
```

Comment:
The whole of Step2 is now redundant. These yb-admin commands are not needed anymore.
I guess we will still use this diagram, but that will also need to move into Step 1...
Maybe right after we create tablespaces (and before creating the tables themselves?)

Reply:
And it's gone :)